### DISCUSSION TITLE
Planetscale + Database Strategy

### DISCUSSION THREAD

**Discussion by Rhys97W** [nextauthjs/next-auth#11313] [Help]
Im trying to configure Planetscale to work with the database session strategy on Vercel using Prisma however am falling flat, even with the docs.
Are there any implementation example repos that I can look at to see where I'm going wrong? Or if anyone can share the auth.ts with the Planetscale+Prisma+Database strategy configuration?

**Comment by k-taro56** [comment]
https://github.com/nextauthjs/next-auth-example
